modules/virt-configuring-huge-pages-for-vms.adoc

```diff
@@ -29,6 +29,7 @@ If you edit a running virtual machine, the virtual machine must be rebooted for
 `memory.hugepages.pageSize` parameters to the `spec.domain`. The following configuration snippet is
 for a virtual machine that requests a total of `4Gi` memory with a page size of `1Gi`:
 +
+.Example output
```

Comment:
As in the other example, this is a YAML file that the user is opening in a text editor so it doesn't count as output.

Reply:
Done.